tao/CMakeLists.txt

```diff
@@ -86,12 +86,21 @@ find_package(Protobuf ${Tensorflow_PB_VERSION}
     QUIET
 )
 if(NOT DEFINED Protobuf_PROTOC_EXECUTABLE)
-    tao_execute_cmd('
-        DESC "download protoc"
-        PWD "."
-        COMMAND bash ${tao_ops_SOURCE_DIR}/tools/download_protoc.sh ${CMAKE_CURRENT_BINARY_DIR} ${Tensorflow_PB_VERSION}
-        STD_OUT Protobuf_PROTOC_EXECUTABLE
-    )
+    if(${TAO_AARCH64})
```

Comment:
Just ensure that these will get eliminated once we've transferred to Bazel and protoc are compiled from source. @Orion34-lanbo